src/test/scala/com/microsoft/hyperspace/index/rules/HyperspaceRuleTestSuite.scala

```diff
@@ -51,7 +59,7 @@ trait HyperspaceRuleTestSuite extends HyperspaceSuite {
                 .Columns(indexCols.map(_.name), includedCols.map(_.name)),
               IndexLogEntry.schemaString(schemaFromAttributes(indexCols ++ includedCols: _*)),
               10)),
-          Content(getIndexDataFilesPath(name).toUri.toString, Seq()),
+          Content.fromLeafFiles(Seq(indexFile)),
```

Comment:
nit: does it make sense to test with multiple files (>1) to make sure `FileIndex` is populated correctly? (e.g., ` val location = new InMemoryFileIndex(spark, index.content.files, Map(), None)`)

Reply:
thanks @imback82 , I updated the tests to support checking multiple files. Although as of now I havent added muli-level directory structure in index files. Please let me konw if we can push it for later or need to do it with this PR.

Reply:
keeping this comment unresolved depending on your suggestion